lib/axon/losses.ex

```diff
@@ -1024,4 +1024,170 @@ defmodule Axon.Losses do
       end
     )
   end
+
+  @doc """
+  WIP
+  y_pred should be log_softmax output of shape (B, T, D),
+  y_true should be of shape (B, S),
+  where
+  B - batch size,
+  T - sequence length,
+  D - dictionary size, including blank
+  S - maximum target length
+  y_true nas to be preencoded, its first and last characters should be
+  blank, each initial character should be separated with blanks,
+  double blanks mark the end of target sequence.
+  """
+  defn ctcloss(y_true, y_pred, opts \\ []) do
+    opts = keyword!(opts, blank: 0, reduction: :none)
+    # eps = Nx.tensor([1.0e-7])
```

Comment:
Is this meant to be used?

Reply:
Log(0) could be a thing. What do you think? Should we catch it or should we eps it?

Reply:
I would use eps in that case